openwisp-utils-qa-checks

```diff
@@ -93,17 +96,32 @@ runcheckcommit() {
   fi
 }
 
+runmakemigrationcheck() {
+  echo "Running Make Migrations Check"
+  if [ ! -f "./tests/manage.py" ]; then
+    echo "File manage.py not found"
+  else
+    OUTPUT=$(./tests/manage.py makemigrations --dry-run $MODULE)
```

Comment:
Perhaps, you meant `OUTPUT=$(python ./tests/manage.py makemigrations --dry-run $MODULE)`

Reply:
I don't think its needed

Reply:
adding `python` before is more robust (in case a module does not have the `manage.py` flagged as executable, which may happen)